helmfile/charts/node-local-dns/templates/node-local-dns.yaml

```diff
@@ -197,3 +197,26 @@ spec:
       targetPort: 9253
   selector:
     k8s-app: node-local-dns
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: node-local-dns
+  name: node-local-dns
+  namespace: kube-system
+spec:
+  endpoints:
+  - interval: 30s
+    path: /metrics
+    port: metrics
+    relabelings:
+    - sourceLabels: [__meta_kubernetes_pod_node_name]
+      action: replace
+      targetLabel: instance
```

Comment:
I am replacing the instance IPs with the node names.
let me know if the IPs are more useful or if we want both

Reply:
I didn't know that this was a feature. Is it basically the same as relabel but doing it directly in grafana instead?